src/utils/git.d.ts

```diff
@@ -0,0 +1,355 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
💡 Curious: this whole file is copied over form here: https://github.com/microsoft/vscode/blob/main/extensions/git/src/api/git.d.ts what is the purpose of this please?

Reply:
I looked for NPM Git Vscode packages , nothing exists I guess that's David would have taken this route. But agree we can chat around this :)

Reply:
@hsubramanianaks is correct if you take a look at their [README](https://github.com/microsoft/vscode/tree/main/extensions/git) the intended use pattern is to "Copy src/api/git.d.ts to your extension's sources" so that is what I did. I don't see any packages on npm for it as well